llvm/test/DebugInfo/X86/swift-move-function-debuginfo-async.ll

```diff
@@ -0,0 +1,1895 @@
+;; RUN: llc -O0 -mtriple=x86_64-apple-darwin %s -o %t -filetype=obj
+;; RUN: llvm-dwarfdump --show-children %t | FileCheck --check-prefix=DWARF %s
+
+source_filename = "move_function_dbginfo_async.ll"
```

Comment:
Is ther anything we could do here (llvm-extract?) to make this test shorter? This looks very hard to maintain.